deploy_contract_01.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#******************************************************************************
+#* Filename:            deploy_contract_01.py
```

Comment:
I propose to simplify file headers to remove the possibility of metadata going out of sync. Could we just say
```
This file is part of the Machine Economy Zilliqa Dapp Project.
```
Metadata are already stored in Git. For example, the prototype status is implied by the fact that this file has not been part of a release yet.